js/animate.js

```diff
@@ -0,0 +1,56 @@
+const width =
+  window.innerWidth ||
+  document.documentElement.clientWidth ||
+  document.body.clientWidth;
+const height =
+  window.innerHeight ||
+  document.documentElement.clientHeight ||
+  document.body.clientHeight;
+const constant = width / (height * 2.3);
+
+const randomNumber = (min, max) => {
+  return Math.random() * (max - min) + min;
+};
+
+const fetchx = (n) => {
+  const deg = n * (72 * (Math.PI / 180));
+  return (width / 5) * Math.cos(deg);
+};
+
+const fetchy = (n) => {
+  const deg = n * (72 * (Math.PI / 180));
+  return 160 * Math.sin(deg);
+};
+
+const arr = document.querySelectorAll('.member_animation');
```

Comment:
just a suggestion, can be ignored - could we look for more relatable word here?

Reply:
Changed arr => memberList
I guess this will do. What do you say?

Reply:
Yeah this is better